google-cloud-storage/src/main/java/com/google/cloud/storage/PostPolicyV4.java

```diff
@@ -443,13 +443,21 @@ public String toJson() {
       StringBuilder escapedJson = new StringBuilder();
 
       // Certain characters in a policy must be escaped
-      for (char c : json.toCharArray()) {
+      char[] jsonArray = json.toCharArray();
+      for (int i = 0; i < jsonArray.length; i++) {
+        char c = jsonArray[i];
         if (c >= 128) { // is a unicode character
           escapedJson.append(String.format("\\u%04x", (int) c));
         } else {
           switch (c) {
             case '\\':
-              escapedJson.append("\\\\");
+              // The JsonObject/JsonArray operations above handle quote escapes, so leave any "/""
```

Comment:
is forward slash meant here? Looks like it should be a backslash

Reply:
Should be, will fix